README.md

```diff
@@ -350,10 +350,10 @@ We use simple hand signals during video calls to communicate more efficiently.
 | sign | meaning | context |
 |--- | --- | --- |
 | :fist: | pass (no comment) | moderated discussion |
-| :point_up: | request voice | moderated discussion | 
+| :point_up: | request voice | moderated discussion |
 |:thumbsup:|vote "consent"|S3 decision making|
-|hand with palm down|vote "concern"|S3 decision making|
-|hand with palm up|vote "protest"|S3 decision making|
+|🫳|vote "concern"|S3 decision making|
+|🫴|vote "protest"|S3 decision making|
```

Comment:
This doesn't actually render in github markdown and the change makes it completely unreadable.

Every other change in this PR seems fine.

Reply:
reverted